config/locales/en/errors.yml

```diff
@@ -555,7 +555,7 @@ en:
               blank: Enter the yearly dividend
             will_benefit_from_trust_fund:
               inclusion: Select yes if your client stands to benefit from a trust fund inside or outside the UK
-        steps/capital/residential_property_form:
+        steps/capital/residential_form:
```

Comment:
Line 549 in this file needs to be updated to: `Select which other assets your client owns or part-owns inside or outside the UK`